music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    super(genre, author, label, publish_date)
+    @on_spotify = on_spotify
+    @archived = archived
+  end
+
+  def can_be_archived?()
```

Comment:
- [ ]  Kindly remove empty parenthesis here to implement YAGNI rule in your code.